src/main/java/nl/ordina/jobcrawler/model/Skill.java

```diff
@@ -1,9 +1,18 @@
 package nl.ordina.jobcrawler.model;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import lombok.*;
-
-import javax.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
```

Comment:
instead of 
@Getter
@Setter
@RequiredArgsConstructor
@AllArgsConstructor
@Builder
 we can just use @Data

Reply:
That is indeed true, however when I put ```@Data``` instead of ```@Getter @Setter @RequiredArgsConstructor```, it gives this error:
```
org.hibernate.InstantiationException: No default constructor for entity:  : nl.ordina.jobcrawler.model.Skill
```
So I put ```@Data``` and ```@NoArgsConstructor``` in commit 5678418fa620dfbc238386775c63fb3fbaf3a6b3

Reply:
is this now resolved? Does it work when you solve it like this

Reply:
Yes it is resolved and it works with this solution I mentioned.